frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -794,9 +794,9 @@ export const insightLogic = kea<insightLogicType>([
             },
         ],
         isUsingDataExploration: [
```

Comment:
I assume this is for a follow-up PR?

Reply:
Yep, for now I wanted to make sure this doesn't break self-hosted users who don't have the feature flag added. Will go over all instances of `isUsingDataExploration` and remove them in a follow up PR.

Reply:
Do you mean persisted feature flags? I think that was only so that the feature flag was baked in as enabled, because we don't actually use feature flags for self-hosted setups.